### PR TITLE
ARGO-392 Add recomputations resource description in swagger spec file

### PR DESCRIPTION
**Pull request by pkoro**
Description

 - [x] Add recomputations resource description in swagger spec file

### CODE REVIEW

**Comment by pkoro** [comment]
please review /cc @themiszamani @kaggis

**Comment by kaggis** [comment]
+1

File: doc/swagger/swagger.yaml [hunk 1]
```diff
@@ -1930,6 +2006,49 @@ definitions:
...
+        type: string
+      start_time:
+        type: string
+      end_time:
+        type: string
+      exclude:
+        type: array
+        items:
+          type: string
+      status:
```
Comment (themiszamani) [inline doc/swagger/swagger.yaml]: add a description , different types of statuses.

File: doc/swagger/swagger.yaml [hunk 2]
```diff
@@ -1930,6 +2006,49 @@ definitions:
...
+        type: string
+      requester_email:
+        type: string
+      reason:
+        type: string
+      start_time:
+        type: string
+      end_time:
+        type: string
+      exclude:
```
Comment (themiszamani) [inline doc/swagger/swagger.yaml]: add a description for

**Comment by pkoro** [comment]
@themiszamani descriptions have been added.

**Comment by themiszamani** [comment]
+1

**Pull request merged**